SoomlaiOSProfile/SoomlaProfile.m

```diff
@@ -347,25 +352,25 @@ - (void)like:(Provider)provider andPageId:(NSString *)pageId andReward:(Reward *
 }
 
 - (void)openAppRatingPage {
-
-    NSString* appID = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleIdentifier"];
-    float iOSVersion = [[UIDevice currentDevice].systemVersion floatValue];
-    NSString *reviewURL;
-
-    if (iOSVersion<7) { //IOS < 7.0
-        NSString *templateReviewURLiOS6 = @"itms-apps://ax.itunes.apple.com/WebObjects/MZStore.woa/wa/viewContentsUserReviews?type=Purple+Software&id=APP_ID";
-        reviewURL = [templateReviewURLiOS6 stringByReplacingOccurrencesOfString:@"APP_ID" withString:appID];
-
-    } else { //IOS >= 7.0
-        NSString *templateReviewURLiOS7 = @"itms-apps://itunes.apple.com/app/idAPP_ID";
-        reviewURL = [templateReviewURLiOS7 stringByReplacingOccurrencesOfString:@"APP_ID" withString:[NSString stringWithFormat:@"%@", appID]];
+    NSString *appID = [[NSBundle mainBundle] infoDictionary][@"iTunesAppID"];
+    if (!appID) {
+        [NSException raise:NSInvalidArgumentException format:@"To open app review page, add iTunesAppID at your Info.plist."];
+    } else {
+        SKStoreProductViewController *appPageController = [[SKStoreProductViewController alloc] init];
+        [appPageController loadProductWithParameters:@{
+                SKStoreProductParameterITunesItemIdentifier: appID
+        } completionBlock:nil];
+        appPageController.delegate = self;
+        [[[UIApplication sharedApplication] keyWindow].rootViewController presentViewController:appPageController
+                                                                                       animated:YES completion:nil];
```

Comment:
@EugeneButusov, prefer the following:

1) 
```
if (positive case) {
  // your code
} else /* if (negative case) */ {
  // your code
}
```

OR

2) 
```
if (negative case) {
  // your code
  return; // !!!
}

// your code for positive case
```

(no action item for this at the moment)